ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -6,6 +6,12 @@
 import UIKit
 import SwiftUI
 
+/// Pre-defined styles of cells
+@objc public enum MSFListCellStyle: Int, CaseIterable {
```

Comment:
Do we need this to be publicly exposed and exposed to objc?

Reply:
Also, I believe we don't need this type if we're creating a new subspec for the PersonaCell.
They will be different types.

Reply:
Reactivating this. What's the need for the @objc? And why do we need a new style?